static/docs/src/openapi.yaml

```diff
@@ -283,12 +283,13 @@ paths:
                 $ref: 'pages.yaml#/definitions/page'
               meta:
                 $ref: '#/definitions/meta'
-    put:
+    patch:
       tags:
       - page
-      summary: Set Page
-      description: Sets the provided page.
-      operationId: setPage
+      summary: Update Page
+      description: |
```

Comment:
Is the `|` just so you could put the description all on the next line?

Reply:
Right.  It's especially helpful if the description is going to be multiline